blip-tap.md

```diff
@@ -919,149 +919,251 @@ order to clamp down on price volatility exposure.
 
 The recipient:
 
-  - SHOULD send a `tap_rfq_reject` message if `rfq_id` has been used before
+- SHOULD send a `tap_rfq_reject` message if `rfq_id` has been used before.
+
+- MUST send a `tap_rfq_reject` message if `in_asset_id` is not committed to
+  in the open channel, unless `in_asset_id` is all zeros, representing BTC.
 
-  - MUST send an `tap_rfq_reject` message if `asset_id` is not committed to in
-    the open channel
+- MUST send a `tap_rfq_reject` message if `out_asset_id` is not committed to
+  in the open channel, unless `out_asset_id` is all zeros, representing BTC.
 
-  - MUST send an `tap_rfq_reject` message if the requested `asset_amt` is
-    greater than the settled remote balance of that asset
+- MUST send a `tap_rfq_reject` message if both `in_asset_id` and
+  `out_asset_id` are all zeros, representing BTC, unless either
+  `in_asset_group_key` or `out_asset_group_key` is set.
 
-  - SHOULD take the `suggested_rate_tick` values into account when deciding
-    whether to accept or reject the quote
+- MUST send a `tap_rfq_reject` message if the requested
+  `max_transferable_in_asset` is greater than the settled remote balance of
+  that asset.
+
+- SHOULD take the values of `expiry`, `in_asset_to_btc`, `out_asset_to_btc`, and
+  `max_transferable_in_asset` into account when deciding whether to accept or
+  reject the quote request.
 
 #### Request For Quote Response (`tap_rfq_accept`) + (`tap_rfq_reject`)
 
 Once the edge node has received, the `tap_rfq` message, then it should decide
 if it's able to accommodate the quote or not. 
 
-#### Accepting Quotes  (`tap_rfq_accept`) 
-
-If it can, then it should send `tap_rfq_accept` that returns the quote amount
-the edge node is willing to observe to move `N` units of asset `asset_id`:
+#### Accepting Quotes  (`tap_rfq_accept`)
 
-1. type: ?? (`tap_req_accept`)
-2. data:
-     * [`32*byte`:`rfq_id`]
-     * [`BigSize`:`accepted_rate_tick]
-     * [`BigSize`:`expiry`]
-     * [`64*byte`:`rfq_sig`]
+The edge node may optionally respond to the quote request with a
+`tap_rfq_accept` message, indicating the acceptance of the quote request. This
+message has a BOLT 01 wire message type integer of 52885, which falls within the
+custom message range. The associated TLV stream includes the following TLV
+fields:
 
-TODO(roasbeef): tlv err where?
+| TLV Type | Field Name       | Field Type |
+|----------|------------------|------------|
+| 0        | version          | 1*byte     |
+| 1        | rfq_id           | 32*byte    |
+| 2        | expiry           | BigSize    |
+| 3        | rfq_sig          | 64*byte    |
+| 4        | in_asset_to_btc  | FixedPoint |
+| 5        | out_asset_to_btc | FixedPoint |
 
 where:
 
-* `rfq_id` matches the existing `rfq_id` of a set `tap_rfq`
-
-* `accepted_rate_tick` is the proposed rate for the volume unit expressed in
-  the internal unit of a `tick`.
-
+* `version` is the version of the quote accept message. The current version is
+  `0`.
```

Comment:
I think we'll bump the version to 1 with the latest changes?

Reply:
I think that the fields for the accept message haven't changed since the last release. The wire message is still on version 0. We check each wire message version separately. So we don't need to bump the version here.

Reply:
TLV int types now use odd/even for optional/mandatory. Bumped version.